packages/workshop-utils/src/epic-api.server.ts

```diff
@@ -12,9 +12,13 @@ import {
 import { cachified, fsCache } from './cache.server.js'
 import { getWorkshopConfig } from './config.server.js'
 import { getAuthInfo, setAuthInfo } from './db.server.js'
+import { logger } from './logger.js'
 import { type Timings } from './timing.server.js'
 import { getErrorMessage } from './utils.js'
 
+// Module-level logger for epic-api operations
+const log = logger('epic-api')
```

Comment:
I think we should make all our logs start with `epic:` as a prefix. Let's rename this to `epic:api`